akka-actor-typed-tests/src/test/scala/akka/actor/typed/ActorContextSpec.scala

```diff
@@ -257,21 +263,136 @@ object ActorContextSpec {
     }
   }
 
+  sealed abstract class Start
```

Comment:
This was inlined from the old `TypedActor` as i didn't re-write this test to use the TestKit